io_scene_niftools/modules/nif_import/property/object/__init__.py

```diff
@@ -58,17 +60,32 @@ def import_extra_datas(self, root_block, b_obj):
             if isinstance(n_extra, NifFormat.NiStringExtraData):
                 # weapon location or attachment position
                 if n_extra.name.decode() == "Prn":
-                    for k, v in PRN_DICT.items():
-                        if v.lower() == n_extra.string_data.decode().lower():
-                            b_obj.niftools.prn_location = k
+                    game = bpy.context.scene.niftools_scene.game
+                    if game in PRN_DICT[next(iter(PRN_DICT))]:
+                        # first check specifically in that game
+                        for slot, game_map in PRN_DICT.items():
+                            if game_map[game].lower() == n_extra.string_data.decode().lower():
+                                b_obj.niftools.prn_location = slot
+                                break
+                    if b_obj.niftools.prn_location == "NONE":
+                        # we didn't find anything, either because the game doesn't have it,
+                        # or we have the wrong game. Check all key, value pairs
+                        for slot, game_map in PRN_DICT.items():
+                            for k, v in game_map:
+                                if v.lower() == n_extra.string_data.decode().lower():
+                                    b_obj.niftools.prn_location = slot
+                                    break
+                            else:
+                                continue
+                            break
                 elif n_extra.name.decode() == "UPB":
                     b_obj.niftools.upb = n_extra.string_data.decode()
             elif isinstance(n_extra, NifFormat.BSXFlags):
                 b_obj.niftools.bsxflags = n_extra.integer_data
             elif isinstance(n_extra, NifFormat.BSInvMarker):
                 b_obj.niftools_bs_invmarker.add()
                 b_obj.niftools_bs_invmarker[0].name = n_extra.name.decode()
-                b_obj.niftools_bs_invmarker[0].bs_inv_x = n_extra.rotation_x
-                b_obj.niftools_bs_invmarker[0].bs_inv_y = n_extra.rotation_y
-                b_obj.niftools_bs_invmarker[0].bs_inv_z = n_extra.rotation_z
+                b_obj.niftools_bs_invmarker[0].bs_inv_x = (-n_extra.rotation_x/1000) % (2*pi)
```

Comment:
modulo intended?

Reply:
Yes. I thought it might be easiest to keep it within the 0-360 degrees range for interpretation by the user (since any rotation outside that range will have the exact same effect as the modulo).

Reply:
Yeah its a nice way to wrap/clamp value.